landing_page/app/controllers/users_controller.rb

```diff
@@ -60,9 +43,9 @@ def update
     end
   end
 
-  # DELETE /users/1
-  # DELETE /users/1.json
   def destroy
+    authorize! :update, @user, :message => "Unable to update user"
```

Comment:
Message should be "unable to destroy user"?

Reply:
Resolved in commit 9bd8da11cba94945e7262cefa9a143ad3b3d93b7